src/Impostor.Api/Events/Game/IGameHostChangedEvent.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Impostor.Api.Net;
+
+namespace Impostor.Api.Events
+{
+    public interface IGameHostChangedEvent : IGameEvent
+    {
+        IClientPlayer PreviousHost { get; }
+
+        IClientPlayer NewHost { get; }
```

Comment:
`NewHost` should be nullable, just like [`Game#Host`](https://github.com/Impostor/Impostor/blob/fe7a641bad9639f7b38e102e8876e3a680bac183/src/Impostor.Api/Games/IGame.cs#L27)

Reply:
Actually, `NewHost` would never be null because method returns before event has a chance to be called, sounds like something that should be changed for consistency?